.github/workflows/feature_deploy.yml

```diff
@@ -37,559 +37,151 @@ jobs:
     - name: Initialise Environment
       run: echo "Setting up ${{ env.environment }} environment"
 
-  #################### 1. PORTAL UI ####################
-  deploy-portal-ui:
+  deploy-webapps:
     needs: deployment-environment
-    name: Portal UI
+    name: Deploy Webapps
     runs-on: [self-hosted, linux, deployment]
     # TODO: FN-1077 NOTE: we don't currently have a `feature` GitHub environment yet so use the `dev` one while we're not doing much.
     # Restore the parameterised version throughout!
     environment: dev
     # environment:
     #   name: ${{ needs.deployment-environment.outputs.environment }}
-    env:
-      repository: portal-ui
-      
+    strategy:
+      max-parallel: 4
```

Comment:
I've tested a deploy with this (and making the functionapps dependent on the webapp job) and it succeeds.
The code's a lot neater too.

I considered combining the functionapps with the webapps, which wouldn't be especially difficult, but with them going away at some point, it is perhaps overkill.